HW03/Alexey_Zalyotov/lib/homework.rb

```diff
@@ -0,0 +1,22 @@
+class Homework
+  attr_reader :task, :repo
+  attr_accessor :solution, :status, :review_comment
+
+  def initialize(task:, repo:)
+    @task = task
```

Comment:
Maybe? Or for example `name/id/number`, you just need to have an identifier for this task 
```suggestion
    @task_body = task_body
```

Reply:
Done